### PR TITLE
Add Thai Translation

### PR DESCRIPTION
**Pull request by tomzt**
- Create Folder For Thai 
- Translate two file in to Thai Language

### CODE REVIEW

**Comment by tomzt** [comment]
Hi, Maintainers 
Any suggestions to fix it?

**Comment by JakobLichterfeld** [comment]
Thanks for your change!

**Comment by JakobLichterfeld** [comment]
The build fails due to invalid symbols, so we can not merge this in this state.

File: priv/gettext/th/LC_MESSAGES/default.po [hunk 1]
```diff
@@ -0,0 +1,521 @@
...
+msgid "For more information, see the updated installation guides on %{link}"
+msgstr "สำหรับข้อมูลเพิ่มเติม โปรดดูคู่มือการติดตั้งที่อัปเดตที่ %{link}"
+
+#: lib/teslamate_web/templates/layout/root.html.heex:84
+msgid "The automatically generated encryption key used for the current session can be found <strong>in the application logs</strong>."
+msgstr "คีย์การเข้ารหัสที่สร้างขึ้นโดยอัตโนมัติที่ใช้สำหรับเซสชันปัจจุบันสามารถพบได้<strong>ในบันทึกแอปพลิเคชัน</strong>"
+
+#: lib/teslamate_web/templates/layout/root.html.heex:83
+msgid "To ensure that your <strong>Tesla API tokens are stored securely</strong>, an encryption key must be provided to TeslaMate via the <code>ENCRYPTION_KEY</code> environment variable. Otherwise, a <strong>login will be required after every restart</strong>."
+msgstr เพื่อให้แน่ใจว่า <strong>โทเค็น Tesla API ของคุณจะถูกเก็บไว้อย่างปลอดภัย</strong> จะต้องมอบคีย์การเข้ารหัสให้กับ TeslaMate ผ่านทางตัวแปรสภาพแวดล้อม <code>ENCRYPTION_KEY</code> มิฉะนั้น <strong>จะต้องเข้าสู่ระบบหลังจากรีสตาร์ททุกครั้ง</strong>"
```
Comment (JakobLichterfeld) [inline priv/gettext/th/LC_MESSAGES/default.po:517]: This needs to be "เพื่อให้แน่ใจว่า...

**Review by JakobLichterfeld — CHANGES_REQUESTED** [review]

File: priv/gettext/th/LC_MESSAGES/default.po (hunk 1 above)
Comment (tomzt) [reply priv/gettext/th/LC_MESSAGES/default.po]: completed

**Comment by tomzt** [comment]
@JakobLichterfeld Any suggestions ?

File: priv/gettext/th/LC_MESSAGES/default.po [hunk 2]
```diff
@@ -0,0 +1,521 @@
...
+#: lib/teslamate_web/live/geofence_live/form.html.heex:66
+msgid "Per kWh"
+msgstr "ต่อกิโลวัตต์ชั่วโมง"
+
+#: lib/teslamate_web/live/charge_live/cost.html.heex:110
+msgid "Total"
+msgstr "ทั้งหมด"
+
+#: lib/teslamate_web/live/geofence_live/form.html.heex:125
+msgid "There is <strong>%{n} charging session</strong> at this location for which no costs have been added yet."
```
Comment (JakobLichterfeld) [inline priv/gettext/th/LC_MESSAGES/default.po:416]: This is not correct

**Review by JakobLichterfeld — CHANGES_REQUESTED** [review]

**Comment by tomzt** [comment]
@JakobLichterfeld review again, Thanks

**Comment by JakobLichterfeld** [comment]
Perfect, ty!

**Pull request merged**